_docs/whatiscloud.md

```diff
@@ -0,0 +1,45 @@
+
+There are two kinds of offerings that you might see when browsing a cloud provider’s website. Briefly, these are **Virtual Machines** and **Services**.
+
+* Virtual Machines, or VMs, are exactly what they sound like. They’re a virtual environment that emulates a real computer, hosting an Operating System and any software you might want to run. For example, when using a VM as a server, you need to setup packages, configure settings, and run your server application manually, just as you would if you setup your own computer as a server. Cloud VM offerings include Amazon’s AWS EC2 and Azure’s VMs. VMs won’t be covered in detail here. For now, just know that they exist and what they’re for.
+
+* Services, on the other hand, abstract away the nitty-gritty implementation details. Services allow you to build and deploy your application without worrying about the underlying details. For example, Amazon’s AWS Lambda service allows you to deploy functions that contain regular code. AWS might run your code in a VM, or in a Container. The key thing to remember here is, *it doesn’t matter to you*. Most cloud offerings fall into the Services category, and they make developing cloud applications much easier than it might appear.
+
+### Intriguing...so what can I do in the cloud?
+Almost anything you can dream of. Cloud providers are constantly improving and adding to their offerings. A good place to start would be to take a simple application that you could run locally, such as an app that retrieves and stores information in a database and move it into the cloud. Refer to [LINK HERE] for a few concrete examples.
+
+### Wait, there's more than one cloud provider?
+You bet. Amazon Web Services (AWS), Microsoft Azure, and Google Cloud Platform (GCP) are the three most well-known cloud providers out there today, but they aren’t the only ones. In order to keep these docs coherent (and us, sane) we’ll be focusing on these three.
+
+
+### Why should I care about the cloud? It works fine on my machine!
+Try shipping your machine to the customer, then get back to me. Really though, it’s because the applications that you can create in the cloud can be far more capable than what you can create on your machine. Web hosting, machine learning, databases, serverless functions and more are at your fingertips allowing you to build applications in a way you never could before.
+
+### Alright, you've convinced me. So how do I get started?
+Head to [LINK HERE] and follow the exquisitely detailed instructions to learn how to signup for a free account on each cloud platform. It is highly recommended to peruse the documentation for each offering you want to use. Links to AWS, Azure, and GCP documentation websites have been provided for your convenience.
+* AWS Documentation [LINK]
+* Azure Documentation [LINK]
+* GCP Documentation [LINK]
```

Comment:
To do: External links. We can revisit this as the docs pages get more fleshed out.